app/scripts/lib/persistData.js

```diff
@@ -0,0 +1,27 @@
+import log from 'loglevel';
```

Comment:
Our filename convention is to use kebab-case rather than camelCase. The same conventions as for npm packages.

Reply:
fixed [here](https://github.com/MetaMask/metamask-extension/pull/15978/commits/c01123238c0aaafd150dc7fd6caa72770418d4c8)